src/ts/components/titlebar.ts

```diff
@@ -27,8 +28,16 @@ export class TitleBar extends Container<TitleBarConfig> {
       cssClass: 'ui-titlebar',
       hidden: true,
       components: [
-        new MetadataLabel({ content: MetadataLabelContent.Title }),
-        new MetadataLabel({ content: MetadataLabelContent.Description }),
+        new Container({
```

Comment:
The `TitleBar` is already a `Container`, why is a `Container` within a `Container` necessary? This looks like a hack.

Reply:
This allowed me to apply css similar to the `controlbar` also for the `titlebar`. 
This `css` expect the first level of `.#{$prefix}-ui-container` to represent a 'row' in the UI. Maybe it's no good approach.